ecs-cli/modules/cli/configure/configure.go

```diff
@@ -16,78 +16,119 @@ package configure
 import (
 	"fmt"
 
-	"github.com/Sirupsen/logrus"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
+	"github.com/pkg/errors"
 	"github.com/urfave/cli"
 )
 
-// Configure is the callback for ConfigureCommand.
-func Configure(context *cli.Context) {
-	ecsConfig, err := createECSConfigFromCLI(context)
-	if err != nil {
-		logrus.WithFields(logrus.Fields{
-			"error": err,
-		}).Error("Error saving config.")
-		return
+func fieldEmpty(flag string, context *cli.Context) error {
+	field := context.String(flag)
+	if field == "" {
+		return fmt.Errorf("%s can not be empty.", flag)
+	}
+	return nil
+}
+
+// Cluster is the callback for ConfigureCommand (cluster).
+func Cluster(context *cli.Context) error {
+	if err := fieldEmpty(command.RegionFlag, context); err != nil {
```

Comment:
I can change it to 
```
region := context.String(command.RegionFlag)
if err := fieldEmpty(region, command.RegionFlag); err != nil {
}
```
`fieldEmpty` needs to have the flag name passed in as well, otherwise it can't give a useful error message telling which flag was empty.

Reply:
ahh I see! Sounds good